tests/test_hsm.py

```diff
@@ -781,6 +781,30 @@ def test_bounds_centroid():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_ksb_sig():
```

Comment:
The test doesn't seem all that helpful, in the sense that (for example) if the code was accidentally disabled by some bug, you would still get the right answer.  I recommend doing a test that would make sure the new sig parameters are actually being used.  For example: make a two-component galaxy such that there is a clear ellipticity gradient.  Measure shear using two different effective radii, and require that difference in the shears has the expected sign and is nonzero.